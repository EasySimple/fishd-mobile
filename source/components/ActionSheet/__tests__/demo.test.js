import webDemoTest from '../../../tests/demoTest';

webDemoTest('ActionSheet');
