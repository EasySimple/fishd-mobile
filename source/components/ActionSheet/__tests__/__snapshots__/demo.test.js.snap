// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders ActionSheet demo correctly 1`] = `
<div>
  <div
    class="components-actionSheet-demo-basic demo-card"
  >
    <a
      aria-disabled="false"
      class="fm-button fm-button-primary fm-button-normal"
      role="button"
    >
      <span>
        基础用法
      </span>
    </a>
  </div>
  <div
    class="components-actionSheet-demo-basic demo-card"
  >
    <a
      aria-disabled="false"
      class="fm-button fm-button-primary fm-button-normal"
      role="button"
    >
      <span>
        单行分享
      </span>
    </a>
  </div>
  <div
    class="components-actionSheet-demo-basic demo-card"
  >
    <a
      aria-disabled="false"
      class="fm-button fm-button-primary fm-button-normal"
      role="button"
    >
      <span>
        多行分享
      </span>
    </a>
  </div>
</div>
`;
